Source/Events/OnAuctionHouseClosed.lua

```diff
@@ -1,6 +1,4 @@
 function Auctionator.Events.OnAuctionHouseClosed()
-  print("In OnAuctionHouseClosed")
```

Comment:
Can we change this to `Auctionator.Debug.Message("Auctionator.Events. OnAuctionHouseClosed")`. Not sure why I made this a print statement.